ViewGenerator/tools/webpack/Rules/TypeScript.ts

```diff
@@ -9,7 +9,8 @@ import { getCurrentDirectory } from "../Plugins/Utils";
 const getTypeScriptRuleSet = (useCache: boolean): RuleSetRule => {
 
     const TypeScriptRule: RuleSetRule = {
-        test: /\.tsx?$/
+        test: /\.tsx?$/,
+        exclude: /tests/
```

Comment:
when doing tests in Studio, please confirm we are not doing type checking for tests also.
This file is intended for JS generation. Type checking is done separately by ForkTsCheckingPlugin or something like that... But it takes tsconfig.json as input, we have to exclude there

Reply:
thats great advice! thanks a lot. I will look at that :D